### DISCUSSION TITLE
How do I choose which products to scan for?

### DISCUSSION THREAD

**Discussion by lufusol** [jef/streetmerchant#2489] [Help] [answered]
I still can't believe this isn't configurable in dotenv or explained in the main project page or installation guides, as selecting which products to scan for seems like a crucial feature and the first thing a user is going to want to do when they get started.

Can someone please explain how it's possible to omit scanning for unwanted products?

**Comment by tkoeck** [comment, marked as answer]
You scan for products you want. E.g.

SHOW_ONLY_BRANDS=amd,sony,microsoft
SHOW_ONLY_MODELS=
SHOW_ONLY_SERIES=rx6900xt,rx6800xt,sonyps5c

And of course you set up the shops you want to scan.

There is no black list.

**Reply by lufusol** [reply]
Thank you, the documentation is sparse and vague but I still felt stupid asking this question and I wonder how it seems so obvious to everyone else that it didn't warrant explanation anywhere in the official documentation or the wiki installation guides.  None of those arguments are mentioned anywhere.

What file do those arguments belong in?

What are the valid values for series or where can I find or edit them??

Thanks

**Reply by tkoeck** [reply]
This will all be done in the dotenv (docker) file. You can copy the example file like in the Getting Started section in the main readme file.

The parameters are described in the docs directory of the project. Shouldn't be that hard to set it up. I needed half an hour for it.

The readme files could certainly be improved but keep in mind it's a community project. Everybody who knows more can modify the files and let them merge back to the project.

**Reply by lufusol** [reply]
Wow, ugh, I did not even think to dig deeper into docs, after seeing the same info in the faq and getting-started as the website.  But the files in docs/help and docs/reference look to have the answers.

Thanks again
